pom.xml

```diff
@@ -4,7 +4,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.opentracing.contrib</groupId>
-  <artifactId>opentracing-vertx-web-parent</artifactId>
+  <artifactId>opentracing-vertx-parent</artifactId>
```

Comment:
As the repo is `java-vertx-web`, this should probably stay `opentracing-vertx-web-parent` to avoid conflict with other vertx repo later - or will this repo be renamed to `java-vertx`?